From 1dcf875effe050013602ebb61a108ce5f19e0459 Mon Sep 17 00:00:00 2001
From: Joe Perches <joe@perches.com>
Date: Thu, 13 Mar 2014 05:19:19 +0000
Subject: [PATCH v2 016/021] igbvf: Convert uses of __constant_<foo> to <foo>

Git-commit: 1dcf875effe050013602ebb61a108ce5f19e0459
Patch-mainline: v3.15-rc1
Reference: fate#317398, bsc#909493
Target: sle11-sp4

The use of __constant_<foo> has been unnecessary for quite awhile now.

Make these uses consistent with the rest of the kernel.

Signed-off-by: Joe Perches <joe@perches.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>

---
 drivers/net/ethernet/intel/igbvf/netdev.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: linux-3.0-SLE11-SP4/drivers/net/ethernet/intel/igbvf/netdev.c
===================================================================
--- linux-3.0-SLE11-SP4.orig/drivers/net/ethernet/intel/igbvf/netdev.c
+++ linux-3.0-SLE11-SP4/drivers/net/ethernet/intel/igbvf/netdev.c
@@ -2007,12 +2007,12 @@ static inline bool igbvf_tx_csum(struct
 
 		if (skb->ip_summed == CHECKSUM_PARTIAL) {
 			switch (skb->protocol) {
-			case __constant_htons(ETH_P_IP):
+			case htons(ETH_P_IP):
 				tu_cmd |= E1000_ADVTXD_TUCMD_IPV4;
 				if (ip_hdr(skb)->protocol == IPPROTO_TCP)
 					tu_cmd |= E1000_ADVTXD_TUCMD_L4T_TCP;
 				break;
-			case __constant_htons(ETH_P_IPV6):
+			case htons(ETH_P_IPV6):
 				if (ipv6_hdr(skb)->nexthdr == IPPROTO_TCP)
 					tu_cmd |= E1000_ADVTXD_TUCMD_L4T_TCP;
 				break;
